ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/UpdatedMetaDataTracker.java

```diff
@@ -110,6 +110,10 @@ public void set(String replState, String dbName, String tableName, Map <String,
     if (dbName == null) {
       throw new SemanticException("db name can not be null");
     }
+    if (dbName.contains(".") || ((tableName != null)  && tableName.contains("."))) {
+      throw new SemanticException("Invalid DB or table name." + dbName + "::"
```

Comment:
"::" can be printed if table name is not null

Reply:
This is fine.